runtime/jcl/common/mgmtmemory.c

```diff
@@ -702,6 +703,59 @@ Java_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificati
 				return;
 			}
 
+			/* error tracepoint for the cases
+				the value of init or max is negative but not -1; or
+				the value of used or committed is negative; or
+				used is greater than the value of committed; or
+				committed is greater than the value of max if max is not -1.
+			*/
+			if (TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsageError
+				|| TrcEnabled_Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsage) {
+				const char *nsGcName = (*env)->GetStringUTFChars(env, gcName, NULL);
+				for (idx = 0; idx < gcInfo->arraySize; ++idx) {
+					U_64 preUsed = gcInfo->preUsed[idx];
+					U_64 preCommitted = gcInfo->preCommitted[idx];
+					I_64 preMax = I_64 gcInfo->preMax[idx];
+					U_64 postUsed = gcInfo->postUsed[idx];
+					U_64 postCommitted = gcInfo->postCommitted[idx];
+					I_64 postMax = I_64 gcInfo->postMax[idx];
+					I_64 initialSize = gcInfo->initialSize[idx];
+					const char *nsPoolName = (mgmt->supportedMemoryPools > idx)
+													? mgmt->memoryPools[idx].name
+													: mgmt->nonHeapMemoryPools[idx - mgmt->supportedMemoryPools].name;
+
+					if ((preUsed > preCommitted)
+						|| ((-1 != preMax) && (preCommitted > preMax))
+						|| ((-1 != preMax) && (0 > preMax))
+						|| (postUsed > postCommitted)
+						|| ((-1 != postMax) && (postCommitted > postMax))
+						|| ((-1 != postMax) && (0 > postMax))
+						|| ((-1 != initialSize) && (0 > initialSize))) {
+						Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsageError(env,
+																																	nsGcName,
+																																	nsPoolName,
+																																	initialSize,
+																																	preUsed,
+																																	preCommitted,
+																																	preMax,
+																																	postUsed,
+																																	postCommitted,
+																																	postMax);
+					}
+					Trc_JCL_com_ibm_lang_management_internal_MemoryNotificationThread_processNotificationLoop_memoryUsage(env,
+																															nsGcName,
+																															nsPoolName,
+																															initialSize,
+																															preUsed,
+																															preCommitted,
+																															preMax,
+																															postUsed,
+																															postCommitted,
+																															postMax);
```

Comment:
The first argument goes on the next line (indented one extra tab is sufficient).
Please refer to https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#function-parameter-list-does-not-fit-on-one-line.